xdsl/utils/test_value.py

```diff
@@ -5,3 +5,10 @@ class TestSSAValue(SSAValue):
     @property
     def owner(self) -> Operation | Block:
         assert False, "Attempting to get the owner of a `TestSSAValue`"
+
+    def __eq__(self, other: object) -> bool:
+        return self is other
+
+    # This might be problematic, as the superclass is not hashable ...
```

Comment:
this is copied from another file, I still stand by it, I think it's worth keeping in this file also